js/trash.js

```diff
@@ -145,8 +145,10 @@ class Trashcan {
      * @returns {void}
      */
     updateContainerPosition() {
-        this._container.x = (window.innerWidth / this._scale - Trashcan.TRASHWIDTH) / 2;
-        this._container.y = window.innerHeight / this._scale - Trashcan.TRASHHEIGHT;
+        const offsetFromBottom = 75;
```

Comment:
Should these offsets be based on the sizes/positions of the buttons and the trashcan itself rather than hardcoded? Otherewise, it looks great.

Reply:
I will find surely find a way to avoid hardcoding of variables and will commit soon. Thanks

Reply:
Any progress on this?

Reply:
Yes sir. Will make updated commit soon.